src/Documentation/tutorials_and_samples/tutorial_1.md

```diff
@@ -99,7 +99,7 @@ namespace OrleansBasics
             this.logger = logger;
         }
 
-        Task<string> IHello.SayHello(string greeting)
+        public Task<string> IHello.SayHello(string greeting)
```

Comment:
Because this is an explicit interface implementation (maybe it was to show that that's where the method is defined for the purpose of the tutorial), it cannot have an accessibility modifier

Reply:
This isn't in the interface.  It's the method implementation.  Without the public, it won't compile.

Reply:
![image](https://user-images.githubusercontent.com/1684189/69474748-739b8f80-0d79-11ea-81f1-2c4803654166.png)

Reply:
I'm referring to the `IHello.` text before `SayHello` in the original code:

``` C#
Task<string> IHello.SayHello(string greeting)
```

If you add `public` then you also must remove `IHello.`, since explicit interface implementations cannot have access modifiers.

Valid:
``` C#
Task<string> IHello.SayHello(string greeting)
```

Valid:
``` C#
public Task<string> SayHello(string greeting)
```

Invalid:
``` C#
public Task<string> IHello.SayHello(string greeting)
```

Reply:
Ah, I learned something new today.  I didn't even notice the IHello prefix to SayHello.